src/dataflow/merge/script.py

```diff
@@ -8,11 +8,13 @@
 ### VIASH START
 par = {
     "input": [
-        "./resources_test/merge/pbmc_1k_protein_v3_filtered_feature_bc_matrix_rna.h5mu",
-        "./resources_test/merge/pbmc_1k_protein_v3_filtered_feature_bc_matrix_prot.h5mu",
+        "merged.align_query_reference.output_query_prot.h5mu",
+        "merged.knn.output.h5mu",
```

Comment:
I would prefer to keep these paths because these are available in the test resources?
```suggestion
        "./resources_test/merge/pbmc_1k_protein_v3_filtered_feature_bc_matrix_rna.h5mu",
        "./resources_test/merge/pbmc_1k_protein_v3_filtered_feature_bc_matrix_prot.h5mu",
```